guessthenumber/GuessGame.java

```diff
@@ -0,0 +1,39 @@
+package guessthenumber;
+
+import java.util.Random;
+
+public class GuessGame {
+
+    public void startGame() {
+        Random rand = new Random();
+        int computerGeneratedNumber = rand.nextInt(1, 3);
```

Comment:
As an improvement, we can use `Scanner` class to take the input from the user and assign that value to `computerGeneratedNumber`.